src/main/java/ch/njol/skript/expressions/ExprEntities.java

```diff
@@ -69,6 +69,24 @@ public class ExprEntities extends SimpleExpression<Entity> {
 				"[(all [[of] the]|the)] entities of type[s] %entitydatas% [(in|of) ([world[s]] %-worlds%|1¦%-chunks%)]",
 				"[(all [[of] the]|the)] %*entitydatas% (within|[with]in radius) %number% [(block[s]|met(er|re)[s])] (of|around) %location%",
 				"[(all [[of] the]|the)] entities of type[s] %entitydatas% in radius %number% (of|around) %location%");
+
+		ExprLoopValue.registerLoopValueHandler(ExprEntities.class, (source, type) -> {
+			if (!(source.types instanceof Literal<?>))
+				return false;
+			try (LogHandler ignored = new BlockingLogHandler().start()) {
```

Comment:
I'm not sure Skript's java version supports try-with-resources.

Reply:
This is allowed, since we are declaring the resource within the try block. However, with Skript's version of Java we aren't allowed to use an already declared variable.
The following is okay:
```java
try (LogHandler ignored = new BlockingLogHandler().start())
```
The following is not:
```java
LogHandler ignored = new BlockingLogHandler().start()
try (ignored)
```